src/pages/workspace/invoices/WorkspaceInvoicesPage.tsx

```diff
@@ -28,6 +29,7 @@ function WorkspaceInvoicesPage({route}: WorkspaceInvoicesPageProps) {
         >
             {(hasVBA?: boolean, policyID?: string) => (
                 <View style={[styles.mt3, shouldUseNarrowLayout ? styles.workspaceSectionMobile : styles.workspaceSection]}>
+                    {policyID && <WorkspaceInvoiceVBASection policyID={policyID} />}
```

Comment:
We can wrap `WorkspacePageWithSections` with `AccessOrNotFoundWrapper` to avoid giving access of Invoice page to the user where `areInvoicesEnabled` is false.

Reply:
This is out of the scope of this PR.

Reply:
It looks like a new issue to work on.

Reply:
@madmax330, it is a really good catch. I will create an issue.

Reply:
https://github.com/Expensify/App/issues/49842